packages/gatsby/src/schema/types/sort.ts

```diff
@@ -193,7 +193,7 @@ export const getSortInputNestedObjects = ({
   const itc = convertToNestedInputType({
     schemaComposer,
     typeComposer,
-    postfix: `SortInputProp2`,
+    postfix: `SortInput`,
```

Comment:
This is actual code fix that slipped through original PR ( https://github.com/gatsbyjs/gatsby/pull/36708 )